pkg/agent/controller/agent.go

```diff
@@ -136,14 +144,14 @@ func (a *Controller) updateExportedServiceStatus(export *lighthousev2a1.ServiceE
 	return retryErr
 }
 
-func (a *Controller) newClusterServiceInfo(service *corev1.Service, clusterID string) lighthousev1.ClusterServiceInfo {
+func (a *Controller) newClusterServiceInfo(service *corev1.Service, clusterID string) lighthousev2a1.ClusterStatus {
```

Comment:
Rename method name to reflect the new object it returns

Reply:
Done.